ui/app/__init__.py

```diff
@@ -1,19 +1,12 @@
-import os
-import numpy as np
 from flask import Flask
 from config import Config
-from flask_migrate import Migrate
-from flask_sqlalchemy import SQLAlchemy
 from flask_bootstrap import Bootstrap
 from flask_session import Session
 
 app = Flask(__name__)
 app.config.from_object(Config)
-db = SQLAlchemy(app)
```

Comment:
Can readd the DB connection if these are needed. Was unable to start the app with these in.